demo/index.html

```diff
@@ -127,6 +128,12 @@
      */
     demoProps: {
 
+      resources: {
+        type: Array,
+        defaultValue: { en: { YY: "Year", MM: "Month", DD: "Day" } },
```

Comment:
to be consistent with the current demo, wouldn't this need to be: 

`defaultValue: { en: { YYYY: "Year", MM: "Month", DD: "Day" } },`

Reply:
@joshsylvester the date format in current demo **is** `MM/DD/YY`...
https://github.com/predixdesignsystem/px-datetime-common/blob/fe03eafd24f65442ddbf1c842ed60f4f4a62aad9/demo/index.html#L116